From 35713677f883d6e1e5d650fa8f2145e02c12d4da Mon Sep 17 00:00:00 2001
From: Ben Peled <bpeled@marvell.com>
Date: Mon, 26 Apr 2021 10:45:49 +0300
Subject: [PATCH 1513/1921] dt-bindings: pci: change reset to reset controller
 phandle

replace reset bit mask and system controller
with reset controller and reset bit phandle

Signed-off-by: Ben Peled <bpeled@marvell.com>
Change-Id: I2a4db5c7465c4be470dc45673eb8a24679bfa950
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/50566
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 Documentation/devicetree/bindings/pci/pci-armada8k.txt | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/Documentation/devicetree/bindings/pci/pci-armada8k.txt b/Documentation/devicetree/bindings/pci/pci-armada8k.txt
index 19241ea014fa..070d997efe52 100644
--- a/Documentation/devicetree/bindings/pci/pci-armada8k.txt
+++ b/Documentation/devicetree/bindings/pci/pci-armada8k.txt
@@ -24,10 +24,9 @@ Optional properties:
 - phy-names: names of the PHYs corresponding to the number of lanes.
 	Must be "cp0-pcie0-x4-lane0-phy", "cp0-pcie0-x4-lane1-phy" for
 	2 PHYs.
-- marvell,system-controller: address of system controller needed
-	in order to reset MAC used by link-down handle
-- marvell,mac-reset-bit-mask: MAC reset bit of system controller
-	needed in order to reset MAC used by link-down handle
+- resets: phandle reset controller with int reset controller bit.
+	  needed in order to reset MAC used by link-down handle.
+
 
 Example:
 
@@ -49,6 +48,5 @@ Example:
 		interrupts = <GIC_SPI 32 IRQ_TYPE_LEVEL_HIGH>;
 		num-lanes = <1>;
 		clocks = <&cpm_syscon0 1 13>;
-		marvell,system-controller = <&CP11X_LABEL(syscon0)>;
-		marvell,mac-reset-bit-mask = <CP11X_PCIEx_MAC_RESET_BIT_MASK(1)>;
+		resets = <&CP11X_LABEL(pcie_mac_reset) CP11X_PCIEx_MAC_RESET_BIT(0)>;
 	};
-- 
2.31.1

